### ISSUE TITLE
Nordic reader ignores negative magnitudes

### ISSUE THREAD

**Issue by trichter** [obspy/obspy#2906]
and converts these to positive magnitudes.

I am using ObsPy master.

Example header with magnitude -0.9, gets converted to +0.9:

```
2020 0221 0101 34.5 LI60.1933 24.8300 5.29  HEL  9 0.0-0.9LHEL                1
```

Problem is line 241 in `io/nordic/core.py`

```
mag=_float_conv(line[index - 3:index])
```

This is working for me:

```
mag=_float_conv(line[index - 4:index])
```

**Comment by flixha** [comment]
This is/was indeed a bug in the Nordic-reading implementation. Together with support for the New Nordic format this is fixed in #2814 (waiting for merge approval).

**Comment by flixha** [comment]
Now that #2814 is merged into `master`, this is solved and I'll close this if you don't mind.
